Makefile

```diff
@@ -129,7 +129,7 @@ clean:
 	+ time Rscript -e "if(!requireNamespace('mockery', quietly = TRUE)) install.packages('mockery', repos = 'http://cran.rstudio.com', Ncpus = ${NCPUS})"
 	echo `date` > $@
 
-depends_R_pkg = time Rscript -e "devtools::install_deps('$(strip $(1))', threads = ${NCPUS}, dependencies = TRUE);"
+depends_R_pkg = time Rscript -e "devtools::install_deps('$(strip $(1))', Ncpus = ${NCPUS}, dependencies = TRUE);"
```

Comment:
Just to confirm: Are we still OK to call it from devtools, or or do we also need to switch the spelling to `remotes::install_deps`?

Reply:
OOhhhh yeah didn't even check that!

Reply:
hmmm... that doesn't do it